bot/resources/tags/xy-problem.md

```diff
@@ -1,7 +1,7 @@
 **xy-problem**
 
-Asking about your attempted solution rather than your actual problem.
+The XY problem can be summarised as asking about your attempted solution, rather than your actual problem.
```

Comment:
```suggestion
The XY problem can be summarised as asking about your attempted solution rather than your actual problem.
```
This PR is supposed to be about improving grammar but adding a `,` there is invalid grammar.

Reply:
I don't see how adding the `,` makes it invalid grammar. It's just adding a natural pause in speech.